app/views/mortgage_calculator/property_tax_calculator/_next_home_buyer_explanation.html.erb

```diff
@@ -0,0 +1,6 @@
+<p><%= I18n.t("#{i18n_locale_namespace}.how_calculated") %></p>
+<p><%= I18n.t("#{i18n_locale_namespace}.how_calculated_additional") %></p>
+
+<%= render 'bands_table', rates: @rates %>
```

Comment:
Referring to @malcolmbaig 's observation, shouldn't we pass locals up to here as well?

Reply:
@MarcoBgn ♻️

Reply:
Thanks @margOnline